packages/main/src/webComponents/Popover/Popover.stories.tsx

```diff
@@ -5,46 +5,59 @@ import { PlacementType } from '@ui5/webcomponents-react/lib/PlacementType';
 import { Popover } from '@ui5/webcomponents-react/lib/Popover';
 import { PopoverHorizontalAlign } from '@ui5/webcomponents-react/lib/PopoverHorizontalAlign';
 import { PopoverVerticalAlign } from '@ui5/webcomponents-react/lib/PopoverVerticalAlign';
-import React from 'react';
+import React, { useCallback, useRef } from 'react';
 
 export default {
   title: 'UI5 Web Components / Popover',
   component: Popover
 };
 
-export const generatedDefaultStory = () => (
-  <Popover
-    allowTargetOverlap={boolean('allowTargetOverlap', true)}
-    headerText={text('headerText', '')}
-    horizontalAlign={select('horizontalAlign', PopoverHorizontalAlign, PopoverHorizontalAlign.Center)}
-    initialFocus={text('initialFocus', '')}
-    modal={boolean('modal', false)}
-    noArrow={boolean('noArrow', false)}
-    placementType={select('placementType', PlacementType, PlacementType.Right)}
-    verticalAlign={select('verticalAlign', PopoverVerticalAlign, PopoverVerticalAlign.Center)}
-    footer={
-      <div
-        style={{
-          display: 'flex',
-          justifyContent: 'flex-end',
-          alignItems: 'center',
-          width: 'calc(100% - 1rem)',
-          height: '2.5rem'
-        }}
+export const generatedDefaultStory = () => {
+  const popoverRef = useRef();
+  const onButtonClick = useCallback(
+    (e) => {
+      popoverRef.current.open(e.target);
```

Comment:
`open` is undefined. Please use `openBy`.
```suggestion
      popoverRef.current.openBy(e.target);
```